lib/Server.js

```diff
@@ -1,35 +1,78 @@
 'use strict';
 
+const eventbus = require('probo-eventbus');
 const through2 = require('through2');
 
 class Server {
   constructor(options) {
-    this.consumer = options.consumer;
-  }
-  storeBuildInDatabase(build, done) {
-    // Do stuff
-    let error = null;
-    done(null);
-  }
+    this.db = options.db;
+    this.config = options.config;
+
+    this.consumer = new eventbus.plugins['Kafka'].Consumer({
+      group: options.config.eventGroup,
+      topic: options.config.eventTopic,
+      version: options.config.eventVersion,
+    });
 
-  start() {
-    const self = this;
     const commitStreamOptions = {
       autoCommit: true,
       autoCommitIntervalMs: 1000,
       autoCommitMsgCount: 1000,
-    }
-    self.consumer.rawStream.pipe(through2.obj(function(data, enc, cb) {
-      console.log(data);
-      self.storeBuildInDatabase(data.data.build, function(error) {
-        if (error) {
-          freakOut();
-        }
-        cb(null, data);
+    };
+
+    this.commitStream = this.consumer.createCommitStream(commitStreamOptions);
+    this.dbPlugins = options.dbPlugins;
+  }
+
+  start() {
+    const self = this;
+    self.testConnection(function() {
+      self.consumeData();
+    });
+  }
+
+  testConnection(done) {
+    const self = this;
+    console.log('Attempting to connect to DB');
+
+    self.db.test()
+      .then(function(result) {
+        console.log(result.rows, 'RESULT');
+        console.log('Connection successful!');
```

Comment:
Can logs be sent through bunyan and ultimately elasticsearch like the rest of our services?